shank-macro-impl/src/types/resolve_rust_ty.rs

```diff
@@ -302,18 +302,21 @@ fn ident_to_kind(ident: &Ident, arguments: &PathArguments) -> TypeKind {
                                 TypeKind::Composite(Composite::Vec, vec![])
                             }
                         },
-                        "Option" => match resolve_rust_ty(
-                            ty,
-                            RustTypeContext::OptionItem,
-                        ) {
-                            Ok(inner) => TypeKind::Composite(
-                                Composite::Option,
-                                vec![inner],
-                            ),
-                            Err(_) => {
-                                TypeKind::Composite(Composite::Option, vec![])
+                        "COption" | "Option" => {
```

Comment:
This seems unrelated to the PR?

I agree we need to support different option types, but I wanted to first verify that `COption` is indeed serialized the same way as `Option`.
Please remove this for now .. we can address in a separate PR.

I overlooked this change previously due to all the format changes ...

Reply:
Awesome, removed the COption